test/app-build.js

```diff
@@ -79,34 +87,67 @@ describe('kraken:app', function () {
 
         env.NODE_ENV = 'production';
 
-        testutil.run(base, function (err) {
-            if (err) { return done(err); }
-            var build = require('child_process').spawn('npm', ['run', 'all'], { stdio: 'inherit', env: env });
+        testutil.run(base, function(err) {
+            if (err) {
+                return done(err);
+            }
+            var build = require('child_process').spawn('npm', ['run', 'all'], {
+                stdio: 'inherit',
+                env: env
+            });
 
-            build.on('close', function (code) {
+            build.on('close', function(code) {
                 assert.strictEqual(code, 0);
                 done(err);
             });
         });
     });
 
-    it('scaffolded application does not suffer from dll hell with dust-helpers', function (done) {
+    it('scaffolded application does not suffer from dll hell with dust-helpers', function(done) {
```

Comment:
Yeah, bad.  My editor uses auto formatting with options that didn't match the source.  Will fix 'em up in the PR.